modules/creating-a-machine-pool.adoc

```diff
@@ -6,4 +6,4 @@
 [id="creating_a_machine_pool_{context}"]
 = Creating a machine pool
 
-A default machine pool is created when you install a {product-title} (ROSA) cluster. After installation, you can create additional machine pools for your cluster by using {cluster-manager} or the ROSA CLI (`rosa`).
+A default machine pool is created when you install a {product-title} (ROSA) cluster. After installation, you can create additional machine pools for your cluster by using {cluster-manager} or the {product-title} (ROSA) CLI (`rosa`).
```

Comment:
Another case of two "{product-title} ROSA"s in one paragraph - adjust the second one to just "ROSA" so we get "ROSA CLI (`rosa`)" in the end.